src/CMakeLists.txt

```diff
@@ -52,7 +52,11 @@ if(DILITHIUM_ENABLED)
     )
 endif()
 
+# Set up configuration header for optional features.
+configure_file(${MOCOCRW_PROJECT_ROOT}/cmake/in/MoCOCrWConfig.h.in ${CMAKE_BINARY_DIR}/MoCOCrWConfig.h)
```

Comment:
We should put this into a special dir that follows the usual scheme, so mococrw/Config.h.